client/src/components/SystemStatus/SystemStatus.jsx

```diff
@@ -0,0 +1,141 @@
+import React, { useEffect } from 'react';
+import { Route, Switch, Link, useHistory, useLocation } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+
+import { LoadingSpinner, Section, SectionTableWrapper } from '_common';
+import * as ROUTES from '../../constants/routes';
+import { Sidebar, SectionMessage } from '_common';
+import Sysmon from '../SystemMonitor';
+import { SystemStatusQueueTable } from './SystemStatusQueueTable/SystemStatusQueueTable';
+
+import './SystemStatus.global.css';
+import styles from './SystemStatus.module.scss';
+
+const root = `${ROUTES.WORKBENCH}${ROUTES.SYSTEM_STATUS}`;
+
+const SystemStatusSidebar = ({ systemList }) => {
```

Comment:
@wesleyboar Should these sidebar items be centered?

Reply:
They should not. Thanks for mentioning. The Sidebar component assumes icons. I'll fix that via #821.